js/index.js

```diff
@@ -1 +1,118 @@
 // Your code goes here
+// Change color to pink
+const changeColorPink = function (event) {
+  event.target.style.color = 'hotpink';
+  // event.target.style.border = '2px solid hotpink';
+  // event.preventDefault();
+  console.log('I want to change color');
+}
+
+// Change color to aqua
+const changeColorAqua = function (event) {
+  event.target.style.color = 'aqua';
+  console.log('I want to change color');
+}
+
+// Change color to black
+const changeColorBlk = function (event) {
+  event.target.style.color = 'black';
+  console.log('I want to change color');
+}
+
+// Change Navigation Link colors when hovering over
+const navLink = document.querySelectorAll('nav a');
+navLink.forEach(function(link) {
+  link.addEventListener('mouseover', changeColorPink)
```

Comment:
Good use of comments and multiple event listeners on multiple selections! Very efficient!